Base.rte/Activities/MetaFight.lua

```diff
@@ -976,7 +976,8 @@ function MetaFight:UpdateActivity()
 			self:DisableAIs(false, Activity.NOTEAM);
 			self:InitAIs()
 			-- Reset the mouse value and pathfinding so it'll know about the newly placed stuff
-			UInputMan:SetMouseValueMagnitude(0);
```

Comment:
What's this about?

Reply:
It was broken previously, and when I merged into dev, there was seemingly an edit there, so I'm hoping that it works now?

I should really test it, will resolve after

Reply:
Oh right, it was a lua binding problem, I changed it to `UInputMan:SetMouseValueMagnitude(0, -1);` and it seemed to work.